controlplane/src/core/bufservices/PlatformService.ts

```diff
@@ -6302,11 +6302,20 @@ export default function (opts: RouterOptions): Partial<ServiceImpl<typeof Platfo
         const namespaceRepo = new NamespaceRepository(opts.db, authContext.organizationId);
 
         const namespace = await namespaceRepo.byName(req.namespace);
+        if (!namespace) {
```

Comment:
Namespace is optional. Please revert. By default the CLI shows all subgraphs from all namespaces